app/models/coin.rb

```diff
@@ -33,7 +33,7 @@ class Coin < ApplicationRecord
   has_many :calendar_event_coins
   has_many :calendar_events, through: :calendar_event_coins
 
-  validates :name, uniqueness: true, presence: true
+  validates :name, presence: true
```

Comment:
Hm why did the unique validation get removed?

Reply:
There were already violations in the database and as far as I can tell, there's no requirement for it?